### PR TITLE
Sandeep/bot 87/unify websocket

### PR DESCRIPTION
**Pull request by sandeep-deriv**
## Changes
- Documentation Update
- API Unification 
- Remove duplicate API calls
- Subscribe to prerequisite API calls

## Type of change
- Refactoring

### CODE REVIEW

**Review by rupato-deriv — APPROVED** [review]

File: src/components/Header/tab-content.jsx [hunk 1]
```diff
@@ -3,27 +3,31 @@ import PropTypes from 'prop-types';
...
     const isReal = tab === 'real';
-    const token_list = getTokenList();
-    const onChangeAccount = acc => {
-        const account_token = token_list.find(token => token.accountName === acc);
-        if (account_token?.token && acc !== active_account_name) {
-            dispatch(setAccountSwitcherToken(account_token?.token));
+
+    const onChangeAccount = id => {
+        const active_login_id = getActiveLoginId();
+        console.log(id, active_login_id, 'login_id, active_login_id');
```
Comment (shafin-deriv) [inline src/components/Header/tab-content.jsx:20]: May be this console log is unnecessary?

File: src/api-base.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            console.log('Info: Connection to the server was closed, trying to reconnect.');
+            this.init();
+        }
+    };
+}
+
+export default APIBase;
+
+export const api_base = new APIBase();
+window.api_base = api_base;
```
Comment (rupato-deriv) [inline src/api-base.js:132]: can we remove this ? if this is not required @sandeep-deriv

File: src/components/Header/tab-content.jsx [hunk 3]
```diff
@@ -3,27 +3,31 @@ import PropTypes from 'prop-types';
...
     const item_ref = React.useRef([]);
     const isReal = tab === 'real';
-    const token_list = getTokenList();
-    const onChangeAccount = acc => {
-        const account_token = token_list.find(token => token.accountName === acc);
-        if (account_token?.token && acc !== active_account_name) {
-            dispatch(setAccountSwitcherToken(account_token?.token));
+
+    const onChangeAccount = id => {
+        const active_login_id = getActiveLoginId();
```
Comment (rupato-deriv) [inline src/components/Header/tab-content.jsx:19]: console ?

File: src/components/Header/tab-content.jsx (hunk 1 above)
Comment (sandeep-deriv) [reply src/components/Header/tab-content.jsx:20]: It is removed thanks 👍

File: src/api-base.js (hunk 2 above)
Comment (sandeep-deriv) [reply src/api-base.js:132]: yes, it is removed, thanks 👍

File: src/components/Header/tab-content.jsx (hunk 3 above)
Comment (sandeep-deriv) [reply src/components/Header/tab-content.jsx:19]: already removed 👍

**Pull request merged**